src/Tickets/Commerce/Cart.php

```diff
@@ -409,18 +397,18 @@ public function maybe_delete_expired_products() {
 			return;
 		}
 
-		$transient = get_transient( $transient_key );
-
-		// Bail if we have no data to delete.
-		if ( empty( $transient ) ) {
+		// Bail if ET+ is not in place.
+		if ( ! class_exists( Meta_Storage::class ) ) {
```

Comment:
Do you want to switch this to 

`if ( did_action('tec_tickets_plus_attendee_bind_implementations') === 0 ) {`?

Reply:
I'd be up for changing it, but we're also directly using the class we're checking after this statement. So I think it would be safer to keep it as-is.